server/server/models/user.py

```diff
@@ -109,16 +92,81 @@ def __init__(self, email, nickname, password, create_date, location, first_name,
         self.avatar = avatar
         self.banned_to_date = banned_to_date
 
+
+def from_json(cls, data):
+    """ Deserialize User object from JSON """
+    return cls(**data)
+
+
+def model_to_json(sqlalchemy_object):
+    fields_arr = [prop.key for prop in
+                  class_mapper(sqlalchemy_object.__class__).iterate_properties
+                  if isinstance(prop, ColumnProperty)]
+    _dict = {}
+    for key in fields_arr:
+        _dict[key] = str(getattr(sqlalchemy_object, key))
+    return _dict
+
+
 def users_to_json(obj):
     """ Serialize User object to JSON """
-
-    to_serialize = ['id', 'email', 'nickname', 'password', 'create_date', 'location', 'first_name', 'last_name', 'status_id', 'role_id', 'avatar', 'banned_to_date']
+    to_serialize = ['id', 'email', 'nickname', 'password', 'create_date',
+                    'location', 'first_name', 'last_name', 'status_id',
+                    'role_id', 'avatar', 'banned_to_date']
     d = {}
     for attr_name in to_serialize:
-        d[attr_name] = getattr(obj, attr_name)
+        d[attr_name] = str(getattr(obj, attr_name))
     return d
 
-def from_json(cls, data):
-    """ Deserialize User object from JSON """
 
-    return cls(**data)
+def get_all_users():
+    """ Method to get all users from database """
+    qry = get_dbsession().query(User).all()
+    usr_in_json = list(map(lambda x: model_to_json(x), qry))
+    return usr_in_json
+
+
+def create_usr(json_str):
+    """ Method to add user to database """
+    user = User(**json_str)
+    get_dbsession().add(user)
```

Comment:
Better to use request.dbsession. You can get `request` like parameter for this method. When you do this you also don't need to call `transaction.commit()`